docs/source/Knowledge_Base/refundpolicy.rst

```diff
@@ -0,0 +1,53 @@
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Our policy for refunds on lost items.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Why we have enacted this policy
+===============================
+
+Recently, we have been getting a lot of requests for refunds on items in game that might have been lost in a mod bug/glitch, 
```

Comment:
I'd like to see this updated to reflect more modern times. This looks like a direct copy and paste, and that looks bad when we are using the "new" wiki system and just pasting stuff thats on the forums. Maybe we could use terms like "We get a lot of request for refunds..."

Reply:
I didn't want to re-invent the wheel @Column01, since that was the policy we use right now and the wording we use. If @Aiidoneus wants, I can do a bit of said "reinventing" just let me know what you have in mind column.

Reply:
For now, I will commit this. I do agree with @Column01 that this should be updated to more modern terms. Let us save that for another issue.